jmh-benchmarks/README.md

```diff
@@ -0,0 +1,63 @@
+###JMH-Benchmark module
```

Comment:
I could go either way as well, but I'm leaning towards using `jmh-benchmarks` vs. `benchmarks` to re-enforce the fact we only should use JMH for benchmarking.  Having said that if you prefer `benchmarks`, I'm fine with that.